app/pages/project/floating-ips/FloatingIpsPage.tsx

```diff
@@ -161,11 +162,16 @@ export function FloatingIpsPage() {
       <PageHeader>
         <PageTitle icon={<IpGlobal24Icon />}>Floating IPs</PageTitle>
       </PageHeader>
-      <TableActions>
-        <Link to={pb.floatingIpsNew({ project })} className={buttonStyle({ size: 'sm' })}>
+      <TableControls>
+        <TableControlsText>
+          Floating IPs are public IP addresses that can be attached to instances. They allow
+          your instances to be reachable from the internet. Find out more about{' '}
+          <ExternalLink href={links.floatingIpsDocs}>managing floating IPs</ExternalLink>.
+        </TableControlsText>
+        <TableControlsButton onClick={() => navigate(pb.floatingIpsNew({ project }))}>
```

Comment:
this should be a link haha

Reply:
![image](https://github.com/oxidecomputer/console/assets/22547/a45087c2-b005-4feb-94ec-e7237f4568ae)

Reply:
Thank you for spotting that. Will patch that up.